### PR TITLE
IATA air waybill

### PR DESCRIPTION
**Pull request by nissimsan**
Updated IATA Air Waybill, based on IATA Air Waybill Resolution 600a https://www.iata.org/contentassets/e9cb5a72b88f4f68a5cfc572a50b60c9/eacph-european-air-cargo-programme-handbook.pdf

Example based on: 
house-air-waybill-ex

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: packages/traceability-schemas/schemas/ChargeDeclaration.json [hunk 1]
```diff
@@ -28,29 +20,40 @@
...
+        "dueCarrier": {
+            "$comment": "{\"term\": \"dueCarrier\", \"@id\": \"https://schema.org/price\"}",
+            "title": "Due Carrier",
+            "description": "The total disbursements due to carrier. Box 28A and 28B.",
+            "type": "string"
+        },
+        "total": {
+            "$comment": "{\"term\": \"total\", \"@id\": \"https://schema.org/totalPrice\"}",
+            "title": "Total Collect",
+            "description": "The total of all the charges, i.e. weight/volume charge, valuation charge, other charges due carrier and agent and, if applicable, tax. Box 30A and 30B.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/ChargeDeclaration.json:55]:
```suggestion
            "description": "The total of all the charges, i.e., weight/volume charge, valuation charge, other charges due carrier and agent, and, if applicable, tax. Box 30A and 30B.",
```

File: packages/traceability-schemas/schemas/ConsignmentRatingDetail.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    },
+    "commodityItemNumber": {
+      "$comment": "{\"term\": \"commodityItemNumber\", \"@id\": \"https://service.unece.org/trade/uncefact/vocabulary/uncefact/#discountIndicator\"}",
+      "title": "Commodity Item Number",
+      "description": "This box shall be completed as follows: when a specific commodity rate applies, the IATA item description number shall be inserted on the “C” or “U” rate class line; when a class rate reduction applies, the per- centage which is applied to the appropriate charge or rate shall be inserted on the “R” rate class line, preceded by the rate class code to which it refers, e.g. 33% reduction on the normal rate shall be expressed as N67; when a class rate surcharge applies, the per- centage which is applied to the appropriate charge or rate shall be inserted on the “S” rate class line, preceded by the rate class code to which it refers, e.g. 50% surcharge of the minimum charge shall be expressed as M150; when a unit load device rate applies, the ULD rate class type used shall be inserted on the “X” rate class line.Box 22E.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/ConsignmentRatingDetail.json:48]:
```suggestion
      "description": "This box shall be completed as follows: when a specific commodity rate applies, the IATA item description number shall be inserted on the “C” or “U” rate class line; when a class rate reduction applies, the percentage which is applied to the appropriate charge or rate shall be inserted on the “R” rate class line, preceded by the rate class code to which it refers, e.g., 33% reduction on the normal rate shall be expressed as N67; when a class rate surcharge applies, the percentage which is applied to the appropriate charge or rate shall be inserted on the “S” rate class line, preceded by the rate class code to which it refers, e.g., 50% surcharge of the minimum charge shall be expressed as M150; when a unit load device rate applies, the ULD rate class type used shall be inserted on the “X” rate class line. Box 22E.",
```

File: packages/traceability-schemas/schemas/ConsignmentRatingDetail.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      "$comment": "{\"term\": \"grossWeightUnit\", \"@id\": \"https://schema.org/unitCode\"}",
+      "title": "Gross Weight Unit",
+      "description": "22C",
+      "type": "string"
+    },
+    "rateClass": {
+      "$comment": "{\"term\": \"rateClass\", \"@id\": \"https://service.unece.org/trade/uncefact/vocabulary/uncefact/#freightChargeTariffClassCode\"}",
+      "title": "Rate Class",
+      "description": "22D",
+      "enum": ["M—minimum charge", "N—normal rate", "Q—quantity rate", "B—basic charge (optional use)", "K—rate per kilogram (optional use)", "P—international priority service rate", "C—specific commodity rate", "R—class rate reduction", "S—class rate surcharge", "U—unit load device basic charge or rate E—unit load device additional rate X—unit load device additional information Y—unit load device discount", "W—weight increase"]
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/ConsignmentRatingDetail.json:43]:
```suggestion
      "enum": ["M—minimum charge", "N—normal rate", "Q—quantity rate", "B—basic charge (optional use)", "K—rate per kilogram (optional use)", "P—international priority service rate", "C—specific commodity rate", "R—class rate reduction", "S—class rate surcharge", "U—unit load device basic charge or rate", "E—unit load device additional rate", "X—unit load device additional information", "Y—unit load device discount", "W—weight increase"]
```

File: packages/traceability-schemas/schemas/ForeignChargeDeclaration.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        "foreignCurrencyConvertionRate": {
+            "$comment": "{\"term\": \"foreignCurrencyConvertionRate\", \"@id\": \"https://schema.org/currentExchangeRate\"}",
+            "title": "Currency Conversion Rate",
+            "description": "The foreign currency code followed by the conversion rate shall be inserted. Box 33A.",
+            "type": "string"
+        },
+        "foreignChargesCurrency": {
+            "$comment": "{\"term\": \"foreignChargesCurrency\", \"@id\": \"https://schema.org/currency\"}",
+            "title": "Charges in foreign Currency",
+            "description": "The amount shown in the “Total Collect” box shall be inserted after conversion to the foreign currency. Box 33B.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/ForeignChargeDeclaration.json:31]:
```suggestion
            "description": "The amount shown in the \"Total Collect\" box shall be inserted after conversion to the foreign currency. Box 33B.",
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor nits

**Comment by nissimsan** [comment]
Excellent catches, @TallTed, especially the enums - appreciated!

**Review by mkhraisha — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**